bgpd/bgp_evpn_vty.c

```diff
@@ -4085,7 +4085,7 @@ DEFUN(show_bgp_l2vpn_evpn_summary,
  */
 DEFUN(show_bgp_l2vpn_evpn_route,
       show_bgp_l2vpn_evpn_route_cmd,
-      "show bgp l2vpn evpn route [detail] [type <macip|multicast|es|prefix>] [json]",
+      "show bgp l2vpn evpn route [detail] [type <macip|2|multicast|3|es|4|prefix|5>] [json]",
```

Comment:
instead of 2|3|4|5 maybe have it as a number-range or string. i would switch it to DEFPY to do that. @qlyoung thoughts?

Reply:
I would bet for like it is for now because it's trivial change and easy to read. That's only my thoughts.

Reply:
you don't need defpy for that, the syntax is the same if you wanted to do it with a range:
`<macip|multicast|es|prefix|(3-5)>`

what defpy buys you is automatic variable binding:
`<macip|multicast|es|prefix|(3-5)>$type_name`

in the handler function `type_name` is automagically defined for you so you can stop doing manual indexing

that said I think the way it is right now is better because you get a doc string specific to the number, you'd have to write a generic one for the range